devel/libert_util/src/regression.c

```diff
@@ -97,13 +97,27 @@ double regression_unscale(const matrix_type * beta , const matrix_type * X_norm
    beta = inv(X'�X)�X'�y
 */
 
-void regression_OLS( const matrix_type * X , const matrix_type * Y , matrix_type * beta) {
+void regression_OLS( const matrix_type * X , const matrix_type * Y , const matrix_type* E, matrix_type * beta) {
```

Comment:
If we ignore the origin in the `fwd_step_enkf` algorithm - what is the `E` matrix really in terms of this regression function?